packages/image_picker/image_picker_ios/ios/Classes/FLTImagePickerPlugin.m

```diff
@@ -538,25 +572,17 @@ - (void)imagePickerController:(UIImagePickerController *)picker
   }
   if (videoURL != nil) {
     if (@available(iOS 13.0, *)) {
-      NSString *fileName = [videoURL lastPathComponent];
-      NSURL *destination =
-          [NSURL fileURLWithPath:[NSTemporaryDirectory() stringByAppendingPathComponent:fileName]];
-
-      if ([[NSFileManager defaultManager] isReadableFileAtPath:[videoURL path]]) {
-        NSError *error;
-        if (![[videoURL path] isEqualToString:[destination path]]) {
-          [[NSFileManager defaultManager] copyItemAtURL:videoURL toURL:destination error:&error];
-
-          if (error) {
-            [self sendCallResultWithError:[FlutterError
-                                              errorWithCode:@"flutter_image_picker_copy_video_error"
-                                                    message:@"Could not cache the video file."
-                                                    details:nil]];
-            return;
-          }
-        }
-        videoURL = destination;
+      NSError *error;
+      NSURL *destination = [FLTImagePickerPhotoAssetUtil saveVideoFromURL:videoURL error:&error];
+      if (error) {
+        [self sendCallResultWithError:[FlutterError
+                                          errorWithCode:@"flutter_image_picker_copy_video_error"
+                                                message:@"Could not cache the video file."
+                                                details:nil]];
```

Comment:
If you want to use the NSError, you could put the localized description here.

Reply:
Why is this resolved? You're still using `NSError`, but throwing away all of its information.

Reply:
Just a mis-click I think.